test/Trip-test.js

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai';
+const expect = chai.expect;
+
+import Trip from '../src/Trip'
+
+import trips from './trip-data-sample'
+import travelers from './user-data-sample'
+import destinations from './destinations-data-sample'
+
+describe('Trip tests', function() {
+  let trip = new Trip(trips.trips[0], destinations, travelers)
+  it('should have default properties', function() {
+    expect(trip.userID).to.equal(1);
```

Comment:
Good job figuring out how to expect multiple values within the same test.  I didn't know you could do this.  Super awesome!